protocol/flows/ibd/handle_get_blocks.go

```diff
@@ -3,10 +3,13 @@ package ibd
 import (
 	"github.com/kaspanet/kaspad/blockdag"
 	"github.com/kaspanet/kaspad/netadapter/router"
+	"github.com/kaspanet/kaspad/protocol/protocolerrors"
 	"github.com/kaspanet/kaspad/util/daghash"
 	"github.com/kaspanet/kaspad/wire"
 )
 
+const ibdBatchSize = 80
```

Comment:
Didn't we agree 100 is an OK batch size?

Reply:
Also maybe make ibdBatchSize dependant on the size of the route buffers?
(Or at the very least, have a comment that mentions the reason for this)